lib/runner.js

```diff
@@ -46,6 +46,10 @@ var Runner = function(config) {
     flow.timeout(1000, 'waiting for debugger to attach');
   }
 
+  if (config.debuggerServerPort) {
+    process.debuggerServerPort = config.debuggerServerPort;
```

Comment:
Is there a reason that we need to attach this to the `process` object? I know we do that within `protractor.js` for `process.debugPort`, but I don't remember if there's a compelling reason.